suse_caasp

```diff
@@ -260,6 +260,7 @@ if check_rpm "kubernetes-*client" && [ -f /etc/kubernetes/admin.conf ]; then
     log_cmd $OF 'kubectl api-versions'
     log_cmd $OF 'kubectl config view'
     log_cmd $OF 'kubectl -n kube-system get pods -o wide'
+    log_cmd $OF 'kubectl -n kube-system describe pods'
```

Comment:
There is some duplication here between get pods and describe pods, but I guess it would help to have a list of pods before the detail of all pods.

Reply:
Yeah, in a supportutils dump like this it is useful to have lots of information even if it is duplicated. Anything to help a tired support engineer find what they are looking for. :)